pipelines/pipeline.go

```diff
@@ -0,0 +1,84 @@
+package pipelines
+
+import (
+	"encoding/binary"
+	"gopkg.in/DataDog/dd-trace-go.v1/internal/globalconfig"
+	"hash/fnv"
+	"math/rand"
+	"time"
+
+	"gopkg.in/DataDog/dd-trace-go.v1/internal/log"
+)
+
+type Pipeline struct {
+	hash     uint64
+	callTime time.Time
+	service  string
+	edgeName string
```

Comment:
Same question here. Do we need the word `edge`? Does it clash with some other name? Shouldn't `edge` be enough?